default_app/index.html

```diff
@@ -112,62 +136,93 @@
-    </svg>
-
-    <span class="header-version vertical-middle"></span>
-
-    <div class="header-nav vertical-middle">
-      <a href="https://electron.atom.io/docs">Docs</a>
-      <a href="https://github.com/electron/electron">Repository</a>
-      <a href="https://electron.atom.io/blog">Blog</a>
-    </div>
-  </div>
-
-  <div class="container">
-
-    <p>
-    To run your app with Electron, execute the following command in your
-    Console (or Terminal):
-    </p>
-
+    <h2>Running An App</h2>
+    <p>Execute the following command on the command line:</p>
     <pre class="command-example"></pre>
-
     <p>
-    The <code>path-to-your-app</code> should be the path to your own Electron
-    app.
+      The <code>path-to-your-app</code> should be the path to your own Electron app.
     </p>
 
-    <p>You can read the <a class="quick-start-link">quick start</a>
-    guide in Electron's <a class="docs-link">docs</a>
-    to learn how to write one.
-    </p>
-
-    <p>
-    Or you can just drag your app here to run it:
-    </p>
-
-    <div id="holder">
-      Drag your app here to run it
+    <h2>Resources</h2>
+    <div>
+      <div class="linkcol">
+        <h4>Blog</h4>
+        <a class="hero-link" href="https://electron.atom.io/blog">
+          <span class="octicon hero-octicon octicon-gist" style="padding-left:2px" aria-hidden="true"></span>
+        </a>
+      </div>
+      <div class="linkcol">
+        <h4>Repository</h4>
+        <a class="hero-link" href="https://github.com/electron/electron">
+          <span class="octicon hero-octicon octicon-mark-github" style="padding-left:2px" aria-hidden="true"></span>
+        </a>
+      </div>
+      <div class="linkcol">
+        <h4>Docs</h4>
+        <a class="hero-link" href="https://electron.atom.io/docs">
+          <span class="octicon hero-octicon octicon-gear" style="padding-left:2px" aria-hidden="true"></span>
+        </a>
+      </div>
+      <div class="linkcol">
+        <h4>API Demos</h4>
+        <a class="hero-link" href="https://github.com/electron/electron-api-demos">
+          <span class="octicon hero-octicon octicon-star" style="padding-left:2px" aria-hidden="true"></span>
+        </a>
+      </div>
+      <div class="linkcol">
+        <h4>Electron-Forge</h4>
+        <a class="hero-link" href="https://electronforge.io">
+          <span class="octicon hero-octicon octicon-telescope" style="padding-left:2px" aria-hidden="true"></span>
```

Comment:
Too bad there isn't an anvil octicon 😄

Reply:
if you'd prefer a different one, let me know!

Reply:
@MarshallOfSound will have an opinion on this